# Copyright (c) 2017, United States Government, as represented by the
# Administrator of the National Aeronautics and Space Administration.
# 
# All rights reserved.
# 
# The Astrobee platform is licensed under the Apache License, Version 2.0
# (the "License"); you may not use this file except in compliance with the
# License. You may obtain a copy of the License at
# 
#     http://www.apache.org/licenses/LICENSE-2.0
# 
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
# License for the specific language governing permissions and limitations
# under the License.
#
# General parameters needed for the vive system

int32 timebase_hz
int32 timecenter_ticks
int32 pulsedist_max_ticks
int32 pulselength_min_sync
int32 pulse_in_clear_time
int32 pulse_max_for_sweep
int32 pulse_synctime_offset
int32 pulse_synctime_slack